action/protocol/poll/governance_protocol.go

```diff
@@ -81,24 +60,16 @@ func NewGovernanceChainCommitteeProtocol(
 	if err != nil {
 		log.L().Panic("Error when constructing the address of poll protocol", zap.Error(err))
 	}
+
 	return &governanceChainCommitteeProtocol{
-		indexer:                   candidatesIndexer,
-		candidatesByHeight:        candidatesByHeight,
-		getCandidates:             getCandidates,
-		getKickoutList:            getKickoutList,
-		getUnproductiveDelegate:   getUnproductiveDelegate,
 		electionCommittee:         electionCommittee,
 		initGravityChainHeight:    initGravityChainHeight,
 		getBlockTime:              getBlockTime,
-		numCandidateDelegates:     numCandidateDelegates,
-		numDelegates:              numDelegates,
 		addr:                      addr,
 		initialCandidatesInterval: initialCandidatesInterval,
-		productivityByEpoch:       productivityByEpoch,
-		productivityThreshold:     productivityThreshold,
-		kickoutEpochPeriod:        kickoutEpochPeriod,
 		kickoutIntensity:          kickoutIntensity,
```

Comment:
can we entirely remove kickoutIntensity field here by also moving the part related with kick-out in CreateGenesisStates()?